sale_automatic_workflow_stock/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2013 Camptocamp SA (author: Guewen Baconnier)
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    """Extend to add stock related workflow."""
+
+    _inherit = "sale.order"
+
+    # pylint: disable=W8110
+    @api.onchange("workflow_process_id")
+    def _onchange_workflow_process_id(self):
+        """Override to add stock related workflow."""
```

Comment:
Better to use a simple comment to not replace the docstring of the original method
```
# Override to add stock related workflow
```